### PR TITLE
added checking of ptr before it was dereferenced

### PR DESCRIPTION
**Pull request by ProjectMutilation**
Hi! I found potential bug, currentWorksheet() can return nullptr, it follows that the pointer must be checked before it was uses

### CODE REVIEW

**Comment by dantti** [comment]
Can you please fix indentation to look like the rest of the file?

**Comment by ProjectMutilation** [comment]
> Can you please fix indentation to look like the rest of the file?
It is display distortion, indentation is ok in the patch
image

**Comment by ProjectMutilation** [comment]
@dantti if you mean spaces instead tab, i replaced it

File: QXlsx/source/xlsxdocument.cpp [hunk 1]
```diff
@@ -1373,6 +1375,7 @@ bool Document::autosizeColumnWidth(const CellRange &range)
     }
 
     const QMap<int, int> colWidth = getMaximalColumnWidth(range.firstRow(), range.lastRow());
+    if(colWidth.isEmpty()) return false;
```
Comment (dantti) [inline QXlsx/source/xlsxdocument.cpp:1378]: This is not needed

**Review by dantti — COMMENTED** [review]
Actually only the first part of the patch is useful all of the others just duplicate the check

**Comment by ProjectMutilation** [comment]
@dantti now is ok?

**Pull request merged**